static/css/inline-selection.scss

```diff
@@ -253,6 +253,57 @@ $overlay-z-index: 900000;
   z-index: $overlay-z-index;
 }
 
+.pageshot-select-mode-background {
```

Comment:
This all needs to use `@extend .pageshot-reset;`, I'm seeing a lot of style conflicts.  E.g.:

![image](https://cloud.githubusercontent.com/assets/44390/16887192/2a500fdc-4a9e-11e6-9e4f-0fe25f870191.png)

Or:

![image](https://cloud.githubusercontent.com/assets/44390/16887211/450c0704-4a9e-11e6-816c-163ab71ff6d9.png)